### PR TITLE
Add switch fiat currencies screen #5758

### LINKED ISSUE #5758: Add switching between fiat currencies feature

### PR DESCRIPTION
**Pull request by oa-s**
Closes #5758

### CODE REVIEW

File: AlphaWallet/Settings/ChangeCurrency/Coordinator/ChangeCurrencyCoordinator.swift [hunk 1]
```diff
@@ -0,0 +1,51 @@
+//
+//  ChangeCuurencyCoordinator.swift
+//  AlphaWallet
+//
+//  Created by Vladyslav Shepitko on 30.06.2020.
```
Comment (hboon) [inline AlphaWallet/Settings/ChangeCurrency/Coordinator/ChangeCurrencyCoordinator.swift:5]: The header dates are wrong

**Review by hboon — COMMENTED** [review]
Can you explain how this PR and #5855 work together or what they do?

I ran #5857 and when I change the currency from USD to Japanese Yen, Wallet tab, send currency option as well as the graphs are still in USD.

Then I tried #5855, but it crashed with:

> AlphaWalletFoundation/RealmStore.swift:106: Fatal error: 'try!' expression unexpectedly raised an error: Error Domain=io.realm Code=10 "Migration is required due to the following errors:

Maybe I ran them in the wrong order?

**Comment by oa-s** [comment]
#5857 is still wip , have to fix migration

**Comment by oa-s** [comment]
#5855 only adds currencies screen, will tweak a little bit it, and continue with #5857

**Comment by hboon** [comment]
Ah, ok. Let me know what to check then :)

**Comment by oa-s** [comment]
@hboon pr is pretty good for review

**Pull request merged**